docs/_includes/attr-doc.adoc

```diff
@@ -65,16 +68,27 @@ You can also set the base path to images (default: _empty_), icons (default: `./
 
 When you find yourself typing the same text repeatedly, or text that often needs to be updated, consider assigning it to a document attribute and use that instead.
 
+NOTE: If you're used to XML, you'll know document attributes as User Entities.
```

Comment:
> Some of the comments related to the original content that I simply reordered.

Yep, I could tell from the diff what you moved around and what you introduced. My comments pertain to the aggregate result. I appreciate that moving the content around is what helped us find the stuff that doesn't fit. So thanks!